presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadataUpdateHandle.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.spi.ConnectorMetadataUpdateHandle;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+
+import static java.util.Objects.requireNonNull;
+
+public class HiveMetadataUpdateHandle
+        implements ConnectorMetadataUpdateHandle
+{
+    private final UUID requestId;
+    private final String schemaName;
+    private final String schemaTableName;
```

Comment:
What I meant was to use `SchemaTableName` type from SPI

Reply:
oops...didnt know it existed. Made the suggested change.